src/main/java/com/cleancode/tdd/ReadingCounter.java

```diff
@@ -0,0 +1,39 @@
+package com.cleancode.tdd;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class ReadingCounter {
+    int arr[] = {3, 3, 5, 4, 10, 11, 12};
```

Comment:
hardcoded input values is a limitation. It restricts the way code can be tested. Please separate input into a different module.

Reply:
is this comment addressed?